admin/verdaccio.yaml

```diff
@@ -13,11 +13,25 @@
 
 storage: ../storage
 
+publish:
+  allow_offline: false
+
 # A list of other known repositories we can talk to
 uplinks:
   npmjs:
     url: https://registry.npmjs.org/
 packages:
+  '@docusaurus/react-loadable':
+    access: $all
+    publish: $all
+    proxy: npmjs  
+  '@docusaurus/*':
+    access: $all
+    publish: $all
+  '@*/*':
+      access: $all
+      publish: $authenticated
+      proxy: npmjs    
```

Comment:
One more thing, not sure if you want me to do it on this PR, `- {type: stdout, format: pretty, level: http}` using `pretty is extremely expensive (uses internally collars, regex for templating etc) for E2E, I'd recommend `- {type: stdout, format: json, level: warn}` instead to speed up the registry. @Josh-Cena , can I do it on this one?